README.md

```diff
@@ -132,7 +140,7 @@ One of the lines of code in the following snippet will throw an error. Which one
 
 **Your answer:**
 
-> ...
+> I can't tell.
```

Comment:
Line 3. Because we have declared `"use strict";` at the top of the page, we can't use global variables. This includes the `i` that is declared in the for loop.